subsys/random/rand32_ctr_drbg.c

```diff
@@ -150,8 +151,8 @@ int z_impl_sys_csrand_get(void *dst, uint32_t outlen)
 	} else {
 		ret = -EIO;
 	}
-end:
 #endif
+end:
```

Comment:
Have you checked if this gives a warning about an unused label in the case where the label is present but the goto is not?